castervoice/apps/sublime.py

```diff
@@ -159,16 +194,24 @@ class SublimeRule(MergeRule):
             "two": "2",
             "right": "2",
         }),
+        Choice("filetype", {
+            "pie | python": "py",
+            "mark [down]": "md",
+            "tech": "tex",
+            "tommel": "toml",
+        }),
     ]
     defaults = {
+        "nn": None,
         "n2": 1,
         "n3": 1,
+        "filetype": "",
     }
 
 
 #---------------------------------------------------------------------------
 
-context = AppContext(executable="sublime_text", title="Sublime Text")
```

Comment:
why choose the window title over the executable?

Reply:
Well spotted, too much copy pasting!